### DISCUSSION TITLE
네이버 커머스 ID 인증시 노출되는 스토어 관련 문의

### DISCUSSION THREAD

**Discussion by Guwon** [commerce-api-naver/commerce-solutions#84] [묻고 답하기]
안녕하세요.
유니드컴즈 김훈입니다.

고객의 커머스 ID에 연결된 스토어는 총 5개인데요.
( 스마트 스토어 센터 -> 스토어 이동 -> 스토어 선택 )

동일한 계정으로
`네이버 커머스 ID로 인증`시 스토어 선택 화면에서는 두번째 이미지와 같이 3개만 노출이 되고 있습니다.

스토어 선택에 리스트업 되는 스토어가 다른데 저희쪽에서는 확인이 어려워 문의드립니다.
어떻게 하면 `네이버 커머스 ID로 인증`시 `본..` 스토어를 선택 할 수 있을까요?

![image](https://github.com/user-attachments/assets/48ada3c6-aa2c-4773-ac35-2b0d312f0cf9)

![image](https://github.com/user-attachments/assets/a3f48f13-2ea6-4772-b172-2b7889428876)

**Comment by commerce-api-naver** [comment]
안녕하세요. 커머스API 이민석입니다.

커머스솔루션마켓에서의 솔루션 신규 구독은 로그인하신 사용자 분의 '커머스ID' 계정으로 대상 스토어의 [통합매니저] 또는 [주매니저] 권한을 보유한 스토어에 대해서만 구독 신청이 가능합니다.
또는 대상 스토어가 심사중, 심사거부, 보류, 이용정지, 탈퇴신청중, 탈퇴 상태인 경우에도 커머스솔루션 구독 신청은 불가능합니다.
- 관련한 스마트스토어 고객센터 도움말: https://help.sell.smartstore.naver.com/faq/content.help?faqId=12567

스마트스토어센터에서의 스토어 변경은 그 외 권한을 보유하더라도 목록이 뜰 수 있으나 커머스솔루션 구독 신청 과정에서는 [통합매니저], [주매니저] 권한을 보유한 스토어로만 구독 신청이 가능합니다.
마스킹 처리되지 않은 스크린샷으로 문의주신 사용자 분께서 대상 스토어의 적절한 권한을 보유하고 있는지 확인해보실 수 있을까요?

**Reply by commerce-api-naver** [reply]
@Guwon 님, 커머스API 이민석입니다.

혹시 대상 스토어('본...' 스토어)에서 구독하려는 솔루션이 유니드컴즈의 여러 솔루션 중 어느 솔루션일까요?

**Reply by Guwon** [reply]
@commerce-api-naver 
본드팩토리의 상품관리 AI 매니저 입니다.

![image](https://github.com/user-attachments/assets/5f95f56a-41a6-4b02-9aa4-6139c80e561f)

**Reply by commerce-api-naver** [reply]
@Guwon 님, 커머스API 이민석입니다.

문의주신 내용 모두 관련 정보 확인하여 저희 내부적으로 확인을 진행중입니다.
게시해주신 정보 중 사용자 분께서 상시 게시를 원하시지 않을 만한 항목은 게시글 수정으로 삭제해주셔도 좋습니다.

확인 후 다시 한 번 멘션드리겠습니다.

**Reply by Guwon** [reply]
@commerce-api-naver 
안녕하세요.
유니드컴즈 김훈입니다.

혹시 이부분 바로 확인이 어려운 케이스일까요?
고객께서 아직 대기중이셔서 확인차 문의드립니다.

**Reply by commerce-api-naver** [reply]
@Guwon 님 안녕하세요. 커머스API 이민석입니다.

문의주신 내용 내부적으로 확인중에 있으며 빠르면 오늘 오후 늦게, 늦어도 내일(12/13)까지는 답변을 드리도록 하겠습니다.

**Reply by commerce-api-naver** [reply]
@Guwon 님 안녕하세요. 커머스API 이민석입니다.

현재 커머스솔루션 단에서의 특이사항은 발견하지 못하였기에 대상 정보를 제공받는 데이터 부분을 추가적으로 살펴볼 예정입니다. 대상 영역 역시 최대한 빠르게 살펴보고 말씀드리겠습니다.

'판매자 커머스ID 인증' 팝업의 인터페이스에서 스토어 명단 영역은 3개 정도의 스토어가 표시되는 크기인지라 간혹 스크롤을 내릴 경우 정상적으로 보이는 경우가 있습니다.
혹시 대상 사용자 분께서 스크롤을 내려보시도록 가이드해보실 수 있을까요?

**Reply by Guwon** [reply]
@commerce-api-naver 
안녕하세요.
유니드컴즈 김훈입니다.

작일 고객으로 부터 회신 받았고 스크롤 관련된 브라우저 이슈가 맞았습니다.
다른 브라우저 통해서 정상적으로 노출되는 부분 확인하셨습니다.

확인해주셔서 감사합니다.
